scripts/qa_crowbarsetup.sh

```diff
@@ -3647,14 +3647,18 @@ function adapt_dns_for_docker
 
 function glance_image_exists
 {
-    openstack image show "$1" &>/dev/null
+    #due to https://bugs.launchpad.net/python-openstackclient/+bug/1501362
+    #images cannot be found by name over glance v2 api
+    (openstack image show "$1" || openstack --os-image-api-version 1 image show -f shell) &>/dev/null
```

Comment:
In that case rather split it with an if clause, so it is documented why there are two ways, and it can be cleaned up later on:
```
if iscloudver 6minus; then
    openstack image show "$1" &>/dev/null
else
    openstack --os-image-api-version 1 image show -f shell &>/dev/null
fi
```